src/Core/Logging.fs

```diff
@@ -8,58 +8,82 @@ module Logging =
 
     type Level = DBG|INF|WRN|ERR
```

Comment:
I blew right past this, for the sake of consistency might as well change this too